### PR TITLE
Second update User Guide - Add support for HTML response for WFS layer

### PR DESCRIPTION
**Pull request by ElenaGallo**
## Description

Add support for HTML response for WFS layer on [Layer Settings](https://mapstore--10582.org.readthedocs.build/en/10582/user-guide/layer-settings/#feature-info-form) section

### CODE REVIEW

File: docs/user-guide/layer-settings.md [hunk 1]
```diff
@@ -537,6 +537,10 @@ In particular, the user can choose between:
 !!!note
     Without selecting any format here, the [Identify Tool](navigation-toolbar.md#identify-tool) will return the layers information with the format chosen in Map Settings ( in the [Side Toolbar](mapstore-toolbars.md#side-toolbar)). Once a user specifies the information format in layers settings, instead, that format will take precedence over the map settings only for that specific layer.
 
+!!!note
+    MapStore allows users to choose the *Information format* for `WMS`, `CSW`, and `WFS` services.
```
Comment (tdipisa) [inline docs/user-guide/layer-settings.md:541]:
```suggestion
    From the Layer Settings panel, MapStore allows users to choose the *Information format* for `WMS` and `WFS` layer types.
```

File: docs/user-guide/layer-settings.md [hunk 2]
```diff
@@ -537,6 +537,10 @@ In particular, the user can choose between:
...
+    The **Text** option is only available for `WMS` and `CSW` services, while the **HTML** option for a `WFS service` is only available if that option is enable on the GeoServer side. A GeoServer can expose this kind of output when the [`wfs-freemarker-plugin`](https://docs.geoserver.org/main/en/user/community/wfs-freemarker/index.html) is included in the deploy.
```
Comment (tdipisa) [inline docs/user-guide/layer-settings.md:542]:
```suggestion
    The **Text** option is only available for `WMS` layers, while the **HTML** option for `WFS` layers is available only with GeoServer and if the [`wfs-freemarker`](https://docs.geoserver.org/main/en/user/community/wfs-freemarker/index.html) extension is installed on the GeoServer side. 
```

**Review by tdipisa — CHANGES_REQUESTED** [review]

File: docs/user-guide/layer-settings.md [hunk 3]
```diff
@@ -537,6 +537,10 @@ In particular, the user can choose between:
 !!!note
     Without selecting any format here, the [Identify Tool](navigation-toolbar.md#identify-tool) will return the layers information with the format chosen in Map Settings ( in the [Side Toolbar](mapstore-toolbars.md#side-toolbar)). Once a user specifies the information format in layers settings, instead, that format will take precedence over the map settings only for that specific layer.
 
+!!!note
+    From the Layer Settings panel, MapStore allows users to choose the *Information format* for `WMS` and `WFS` layer types.
+    The **Text** option is only available for `WMS` layers, while the **HTML** option for `WFS` layers is available only with GeoServer and if the [`wfs-freemarker`](https://docs.geoserver.org/main/en/user/community/wfs-freemarker/index.html) extension is installed on the GeoServer side.
```
Comment (offtherailz) [inline docs/user-guide/layer-settings.md:542]:
```suggestion
    The **Text** option is only available for `WMS` layers, while the **HTML** option for `WFS` layers is available only with GeoServer and if the [`wfs-freemarker`](https://docs.geoserver.org/main/en/user/community/wfs-freemarker/index.html) extension is installed on the GeoServer side.
```

**Review by tdipisa — APPROVED** [review]

**Comment by tdipisa** [comment]
Thank you @ElenaGallo you can backport to 2024.02.xx.

**Pull request merged**